infer_structcol/run_structcol.py

```diff
@@ -51,8 +118,8 @@ def calc_refl_trans(volume_fraction, radius, thickness, Sample, ntrajectories=60
     # Calculate the effective index of the sample
     sample_index = ri.n_eff(particle_index, matrix_index, volume_fraction)        
     
-    reflectance = []
-    transmittance = []
+    reflectance = np.zeros(len(wavelength))
+    transmittance = np.zeros(len(wavelength))
```

Comment:
wavelength is defined a few lines earlier (previously line 49, now line 119). You can see it if   you expand the code block just above these changes.